packages/block-library/src/column/edit.js

```diff
@@ -61,11 +64,16 @@ function ColumnEdit( {
 				templateLock={ false }
 				renderAppender={
 					hasChildBlocks
-						? undefined
+						? false
 						: () => <InnerBlocks.ButtonBlockAppender />
 				}
+				__experimentalTagName={ Block.div }
```

Comment:
It wasn't introduced here, but it's not clear to me why we limit to a predefined set of tag names, vs. some sort of factor function (`createBlockComponent( tagName )`) or just rendering some element like `<BlockComponent tagName="div" />` here. What if I wanted a `section` block?

Reply:
It's still experimental, but the intention is to add all tags.